src/components/_modal/pages/order/Order.stories.js

```diff
@@ -75,6 +75,36 @@ OrderViaILL.story = merge({}, DEFAULT_STORY_PARAMETERS, {
   },
 });
 
+export function NoUserAgencies() {
+  useMockLoanerInfo({
+    pickUpBranch: "",
+    loans: [],
+    debt: [],
+    agency: [],
+    orders: [],
+  });
+  return (
+    <OrderPageComponentBuilder
+      title="User has no agencies"
+      description="If user logs in with mitid - and has no libraries"
```

Comment:
"If user logs in with MitID - and has no libraries associated with user account"